kakao/src/main/kotlin/com/agoda/kakao/common/views/KBaseView.kt

```diff
@@ -76,6 +79,17 @@ open class KBaseView<out T> : BaseActions, BaseAssertions {
         function(this as T)
     }
 
+    fun configure(configuratorBuilderAction: ConfiguratorBuilder.() -> Unit) {
+        val configuratorBuilder = ConfiguratorBuilder.createWithHistory(KakaoConfigurator.configurator)
+        configuratorBuilderAction.invoke(configuratorBuilder)
+        view = configuratorBuilder.getConfigurator().viewInteractionDelegateFactory
```

Comment:
I think in the builder pattern finalizing function is named `build()`.

Reply:
agree, done